css-text-decor/Overview.bs

```diff
@@ -860,77 +688,29 @@ Emphasis Mark Style: the 'text-emphasis-style' property</h3>
 <h3 id="text-emphasis-color-property">
 Emphasis Mark Color: the 'text-emphasis-color' property</h3>
 
-  <table class="propdef">
-    <tr>
-      <th>Name:
-      <td><dfn>text-emphasis-color</dfn>
-    <tr>
-      <th><a href="#values">Value</a>:
-      <td>&lt;color&gt;
-    <tr>
-      <th>Initial:
-      <td><a href="https://www.w3.org/TR/css3-color/#currentcolor">currentColor</a>
-    <tr>
-      <th>Applies to:
-      <td>all elements
-    <tr>
-      <th>Inherited:
-      <td>yes
-    <tr>
-      <th>Percentages:
-      <td>N/A
-    <tr>
-      <th>Media:
-      <td>visual
-    <tr>
-      <th>Computed&#160;value:
-      <td>as specified
-    <tr>
-      <th>Animatable:
-      <td>as <a href="https://www.w3.org/TR/css3-transitions/#animtype-color">color</a>
-  </table>
+  <pre class="propdef">
+  Name: text-emphasis-color
+  Value: <<color>>
+  Initial: ''currentColor''
+  Inherited: yes
+  Animation type: <a href="https://www.w3.org/TR/css3-transitions/#animtype-color">color</a>
+  </pre>
 
   <p>This property specifies the foreground color of the emphasis marks.</p>
 
   <p class="note">
-    The <dfn value for=text-emphasis-color>currentcolor</dfn> keyword computes to itself
+    The <dfn value for=text-emphasis-color>currentColor</dfn> keyword computes to itself
```

Comment:
All-lowercase is correct (CSS keywords can have any casing you want, but we purposely use all-lowercase for consistency).

Go ahead and remove the `<dfn>` entirely, and instead just autolink the value as `''currentcolor''`.  Dfns don't belong in Notes, and we're not actually defining this value here; it's defined by the Color spec.